src/XRP/issued-currency-client.ts

```diff
@@ -372,4 +373,59 @@ export default class IssuedCurrencyClient {
       wallet,
     )
   }
+
+  public async authorizeTrustline(
+    accountToTrust: string,
+    currencyName: string,
+    wallet: Wallet,
+  ): Promise<TransactionResult> {
+    if (!XrpUtils.isValidXAddress(accountToTrust)) {
+      throw XrpError.xAddressRequired
+    }
+    const classicAddress = XrpUtils.decodeXAddress(accountToTrust)
+    if (!classicAddress) {
+      throw XrpError.xAddressRequired
+    }
+
+    // TODO (tedkalaw): Remove this when ripple-binary-codec supports X-Addresses.
+    const issuerAccountAddress = new AccountAddress()
+    issuerAccountAddress.setAddress(classicAddress.address)
+
+    const currency = new Currency()
+    currency.setName(currencyName)
+
+    const issuedCurrencyAmount = new IssuedCurrencyAmount()
+    issuedCurrencyAmount.setCurrency(currency)
+    issuedCurrencyAmount.setIssuer(issuerAccountAddress)
+
+    // When authorizing a trustline, set it to 0 first.
+    // TODO: Include doc.
+    issuedCurrencyAmount.setValue('0')
```

Comment:
@amiecorso the docs specify that this field is a `string` (https://xrpl.org/trustset.html) but then the docs for authorize trustline say you should set it to `0` (ie, not `'0'`): https://xrpl.org/authorized-trust-lines.html

any thoughts on this?